shared/android/app/src/main/AndroidManifest.xml

```diff
@@ -44,6 +44,12 @@
           <data android:mimeType="audio/*" />
           <data android:mimeType="application/*" />
         </intent-filter>
+        <intent-filter android:label="filter_react_native">
```

Comment:
It seems like the user sees this `label` before they switch into Keybase

> The icon and label that are set in a component's `<intent-filter>` are shown to the user whenever that component is presented as an option to fulfill an intent.

Should this be something like "Sign with Keybase"?

Reply:
Sounds good, switched to "Open with Keybase", since you can do payments as well as signatures.